fgpyo/sam/__init__.py

```diff
@@ -195,6 +197,12 @@
 NO_REF_POS: int = -1
 """The reference position to use to indicate no position in SAM/BAM."""
 
+STRING_PLACEHOLDER: str = "*"
+"""The value to use when a string field's information is unavailable."""
+
+NO_QUERY_QUALITIES: array = qualitystring_to_array(STRING_PLACEHOLDER)
```

Comment:
Please note that `array` is not a generic collection type and so we cannot make this annotation more specific.